core/shared/src/main/scala/krop/route/Param.scala

```diff
@@ -108,21 +108,21 @@ object Param {
       * convert from A to B and B to A.
       */
     def imap[B](f: A => B)(g: B => A): One[B] =
-      One(name, v => parse(v).map(f), b => unparse(g(b)))
+      One(name, parse(_).map(f), g.andThen(unparse))
```

Comment:
nit: Used shorter syntax of placeholder `_` when explicit variable is not necessarily required.

Fixed such instances elsewhere too.